core/jazz_assets/config/global-config.json

```diff
@@ -31,7 +31,10 @@
 		"cloudwatch_event",
 		"swagger_url",
 		"endpoint_url",
-		"dynamodb_stream_arn"
+    "dynamodb_stream",
+    "iam_role",
+    "sqs",
```

Comment:
sqs_queue?

Reply:
As per Satish's comment.

Reply:
sure but i don t agree with it. look at other resources(eg, dynamodb_stream, iam_role)

Reply:
My thought process - 
There aren't any resources under sqs other than queues, while dynamodb, iam etc. have additional resource types (dynamodb tables, dynamodb streams, iam roles, iam users etc.). 

I'm still fine with sqs_queue too.